doc/examples/features_detection/plot_corner.py

```diff
@@ -17,21 +17,24 @@
 from skimage.transform import warp, AffineTransform
 from skimage.draw import ellipse
 
-
+# Sheared checkerboard
 tform = AffineTransform(scale=(1.3, 1.1), rotation=1, shear=0.7,
-                        translation=(210, 50))
-image = warp(data.checkerboard(), tform.inverse, output_shape=(350, 350))
-rr, cc = ellipse(310, 175, 10, 100)
+                        translation=(110, 30))
+image = warp(data.checkerboard()[:90, :90], tform.inverse,
+             output_shape=(200, 310))
+# Ellipse
+rr, cc = ellipse(160, 175, 10, 100)
 image[rr, cc] = 1
-image[180:230, 10:60] = 1
-image[230:280, 60:110] = 1
+# Two squares
+image[30:80, 200:250] = 1
+image[80:130, 250:300] = 1
 
 coords = corner_peaks(corner_harris(image), min_distance=5)
 coords_subpix = corner_subpix(image, coords, window_size=13)
 
 fig, ax = plt.subplots()
 ax.imshow(image, cmap=plt.cm.gray)
-ax.plot(coords[:, 1], coords[:, 0], '.b', markersize=3)
+ax.plot(coords[:, 1], coords[:, 0], 'oc', markersize=6)
```

Comment:
Ha! This is a nice improvement, but it took me a second to translate each letter on that change. Optional for this PR, but I would suggest changing to `color='cyan', marker='o'`

Reply:
In your configuration @jni, `linestyle=None` is needed too (Took me a minute :D ). I just hate mpl api... never consistent :(

Reply:
@sciunto let's remember to always be kind and constructive in our criticism. mpl was a mature library before skimage was even created. It is older than even NumPy. I don't envy someone having to harmonize an API with that much history, and they are definitely working very hard to achieve this... Again, on a purely volunteer, unpaid (directly) basis.

Reply:
you are right, sorry to the mpl guys.